src/dnd-list/types.js

```diff
@@ -40,7 +40,7 @@ export type ListPropsT = {|
   /** Items (labels) to be rendered */
   items: Array<React.Node>,
   /** Handler for when drag and drop is finished and order changed or item is deleted (newIndex would be -1 in that case) */
-  onChange?: ({oldIndex: number, newIndex: number}) => mixed,
+  onChange: ({oldIndex: number, newIndex: number}) => void,
```

Comment:
why switch from mixed?

Reply:
It needs to match the `onChange` signature of `react-movable` which returns `void`.

Reply:
But if you need to do some `await` shenanigans, StatefulList's `onChange` still returns `mixed`.

Reply:
fine with me so that it can interop with the external lib

Reply:
fyi this is technically a breaking change because it restricts the type constraint

Reply:
Good point, I published new flow types so we can keep using mixed.